src/content/app/species-selector/components/species-search-field/SpeciesSearchField.tsx

```diff
@@ -105,21 +105,24 @@ export const SpeciesSearchField = () => {
   const isNotFound = Boolean(matches && matches.length === 0);
 
   return (
-    <AutosuggestSearchField
-      search={selectedItemText || searchText}
-      placeholder="Common or scientific name..."
-      className={styles.speciesSearchFieldWrapper}
-      onChange={onSearchChange}
-      onSelect={onMatchSelected}
-      matchGroups={matchGroups}
-      searchFieldClassName={styles.speciesSearchField}
-      canShowSuggestions={canShowSuggesions}
-      notFound={isNotFound}
-      notFoundText={NOT_FOUND_TEXT}
-      onFocus={() => setIsFocused(true)}
-      onBlur={() => setIsFocused(false)}
-      rightCorner={<RightCorner status={rightCornerStatus} clear={clear} />}
-    />
+    <>
```

Comment:
you no longer need this fragment. Just check out the original file from the dev branch.